.github/scripts/check_version_consistency.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+import os
+import re
+import sys
+from typing import Set, Tuple
+
+
+def find_version_references(directory: str) -> Tuple[Set[str], Set[str]]:
+    openhands_versions = set()
+    runtime_versions = set()
+
+    version_pattern_openhands = re.compile(r'openhands:0\.(\d{2})')
+    version_pattern_runtime = re.compile(r'runtime:0\.(\d{2})')
```

Comment:
```suggestion
    version_pattern_openhands = re.compile(r'openhands:(\d{1})\.(\d{2})')
    version_pattern_runtime = re.compile(r'runtime:(\d{1})\.(\d{2})')
```

to find x.yz versions